tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,66 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2025.
 
+## August 12, 2025
+
+**General changes**
+
+- Rename "TiDB Cloud Serverless" to "**TiDB Cloud Starter**".
+
+    The auto-scaling entry plan is now named "TiDB Cloud Starter" to better reflect its role for new users. All features, pricing, and the free usage quota remain unchanged.
+
+    Starting August 12, 2025 (PDT), your existing Serverless clusters will appear as Starter in the [TiDB Cloud console](https://tidbcloud.com). Your connection strings, endpoints, and data will remain unchanged, so you do not need to make any code changes or schedule downtime.
+
+- TiDB Cloud Essential preview on AliCloud
```

Comment:
Do we need to also add "Starter" here as TiDB Cloud Starter is also in public preview on Alibaba Cloud